packages/manager/src/services/apiKeys/queries.tsx

```diff
@@ -36,7 +36,10 @@ export const useCreateAPIKey = (property: string) => {
 
 // DELETE OPERATIONS
 export const deleteAPIKey = async (dto: TDeleteApiKeyDTO) => {
-  const { data } = await orchestratorReq.delete(`/apikey/${dto.shortKey}`);
+  const { data } = await orchestratorReq.delete(`/apikey`, {
+    data: { propertyIdentifier: `${dto.propertyIdentifier}`, shortKey: `${dto.shortKey}` },
+    headers: { 'Content-Type': 'application/json' }
```

Comment:
The headers is not needed. Kindly remove it as its already added by `orchestratorReq`

Reply:
Also why wrap it like this `${dto.}`. U can simply put it as `dto.`

Reply:
And package.json bump